addons/event_sale/wizard/event_configurator.py

```diff
@@ -22,3 +24,16 @@ def check_event_id(self):
                     _('Invalid ticket choice "%(ticket_name)s" for event "%(event_name)s".'))
         if error_messages:
             raise ValidationError('\n'.join(error_messages))
+
+    @api.depends('product_id')
+    def _compute_available_event_ticket_count(self):
+        for configurator in self:
+            configurator.available_event_ticket_count = self.env['event.event'].search_count(
+                [('event_ticket_ids.product_id', '=', configurator.product_id.id), ('date_end', '>=', fields.Date.today())])
+
+    @api.depends('event_id')
+    def _compute_event_ticket_id(self):
+        for configurator in self:
+            event_ticket_ids = self.env['event.event.ticket'].search([
+                ('event_id', '=', configurator.event_id.id), ('product_id', '=', configurator.product_id.id)])
+            configurator.event_ticket_id = event_ticket_ids if len(event_ticket_ids) == 1 else False
```

Comment:
What is the purpose of this compute? Some kind of magic to automatically select the ticket if there is only one? We usually don't do that in Odoo, seems a bit overkill to avoid a few clicks in some cases.